examples/howto/plot_simulate_mpi_backend.py

```diff
@@ -52,7 +52,7 @@
 weights_ampa = {'L2_pyramidal': 5.4e-5, 'L5_pyramidal': 5.4e-5}
 net.add_bursty_drive(
     'bursty', tstart=50., burst_rate=10, burst_std=20., numspikes=2,
-    spike_isi=10, repeats=10, location='distal', weights_ampa=weights_ampa,
+    spike_isi=10, numsources=10, location='distal', weights_ampa=weights_ampa,
```

Comment:
just to be clear, what happens to `net.gid_ranges` ? Do you have 10 gids when you add a bursty drive?

Reply:
Yes, 10 gids would be created for this bursty drive. This will (hopefully) establish consistent intuition for why the various number of gids are created for each cell type, as well as differentiate between distal and proximal bursty drive spike times when plotting input histograms.